components/brave_wallet/browser/keyring_service_observer_base.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Please update this to 2023, thanks! So does browser/ui/views/toolbar/wallet_button_notification_source.cc and browser/ui/views/toolbar/wallet_button_notification_source.h.